README.md

```diff
@@ -1,55 +1,53 @@
-level-mem
-=========
+# level-mem
 
-<img alt="LevelDB Logo" height="100" src="http://leveldb.org/img/logo.svg">
+> A convenience package bundling `levelup` and `memdown`
 
-**A convenience package bundling LevelUP & MemDOWN**
+[![level badge][level-badge]](https://github.com/level/awesome)
+[![npm](https://img.shields.io/npm/v/level-mem.svg)](https://www.npmjs.com/package/level-mem)
+[![Build Status](https://secure.travis-ci.org/Level/mem.png)](http://travis-ci.org/Level/mem)
+[![david](https://david-dm.org/Level/mem.svg)](https://david-dm.org/level/mem)
+[![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
+[![npm](https://img.shields.io/npm/dm/level-mem.svg)](https://www.npmjs.com/package/level-mem)
 
-[![NPM](https://nodei.co/npm/level-mem.png)](https://nodei.co/npm/level-mem/)
+This is a convenience package that bundles the current release of [`levelup`](https://github.com/level/levelup) and [`memdown`](https://github.com/level/memdown) and exposes `levelup` on its export.
```

Comment:
Maybe merge this sentence into the short description above, it's redundant except for the "and exposes levelup ..".